lib/Cleantalk/Common/Firewall.php

```diff
@@ -245,4 +246,33 @@ private function isWhitelisted($results)
 
         return false;
     }
+
+    public static function addIpToDatabaseAjaxAction()
```

Comment:
If we always set the IP as allowed, I suggest to rename this method to addIpToDBAsAllowedAjaxAction.

Otherwise, we should specify allow/deny flag.